component/espejote-templates/resourcequota-restrictions.jsonnet

```diff
@@ -0,0 +1,71 @@
+local esp = import 'espejote.libsonnet';
+local config = import 'lib/openshift4-local-storage-resourcequota-restriction/config.json';
+
+local context = esp.context();
+
+local generateResourceQuota(vn, namespace) = {
+  apiVersion: 'v1',
+  kind: 'ResourceQuota',
+  metadata: {
+    annotations: {
+      'syn.tools/source': 'https://github.com/appuio/component-openshift4-local-storage.git',
+    },
+    labels: {
+      'app.kubernetes.io/managed-by': 'espejote',
+      'app.kubernetes.io/part-of': 'syn',
+      'app.kubernetes.io/component': 'openshift4-local-storage',
+    },
+    name: 'openshift4-local-storage-restrict-%s' % vn,
+    namespace: namespace.metadata.name,
+  },
+  spec: {
+    hard: {
+      ['%s.storageclass.storage.k8s.io/persistentvolumeclaims' % sc]: '0'
+      for sc in std.objectFields(std.get(config.local_volumes, vn, {}).storage_class_devices)
+    },
+  },
+};
+
+// Reconcile the given namespace.
+local reconcileNamespace(namespace) =
+  // Remove the ResourceQuota if the namespace belongs to a rook-ceph cluster.
+  if std.get(std.get(namespace.metadata, 'labels', {}), 'argocd.argoproj.io/instance', '') == 'rook-ceph' then [
+    esp.markForDelete(generateResourceQuota(vn, namespace))
+    for vn in std.objectFields(config.local_volumes)
+  ]
```

Comment:
Why do we hard-code a rook-ceph exclusion here? The previous implementation worked correctly without hardcoding specific namespace labels in this component. Instead, the target namespace selecto was constructed from field `restricted_to` for each volume class (cf. the deleted `buildLabelSelector()`).